app/styles/app/modules/flash.sass

```diff
@@ -12,7 +12,7 @@
     width: 100%
   li.notice, li.success, li.error
     opacity: 1
-    z-index: 80
+    @include z-index(flashes)
```

Comment:
I’ve put screenshots elsewhere of flashes covering and not covering things.

Many things:

![image](https://user-images.githubusercontent.com/43280/31220373-8902f4cc-a9c0-11e7-91f2-1a17ca41abb8.png)